src/lib/apiClient.ts

```diff
@@ -0,0 +1,28 @@
+
+import { Session } from 'next-auth'
+const API_URL = 'http://localhost:3000/api';
+
+export const apiClient = {
+  publishProfile: async (session: Session | null) => {
```

Comment:
**nitpick:** I would change it to `publishMyProfile` as it's related to only publishing by authorized user. `publishProfile` would fit better if we pass `profileId` as argument instead of `session`